tests/test_converter.py

```diff
@@ -1946,6 +2096,7 @@ def built_pl_circuit():
             )
         )
 
+    # pylint:disable=not-context-manager
```

Comment:
I think you can place this just once at the top of the class, instead of covering every method with it.